### DISCUSSION TITLE
How to return/jump back to the terminal/shell

### DISCUSSION THREAD

**Discussion by Manubi** [sxyazi/yazi#242] [1. Q&A] [answered]
I couldn't find it, but i would enjoy it if there would be a key to jump back to the terminal again instead of interacting with it. Meaning using yazi for navigation. 

also how can I change the $Editor or add another program i'd like to use? thanks!

**Comment by sxyazi** [comment, marked as answer]
> a key to jump back to the terminal again instead of interacting with it

In Yazi, you can return to the terminal/shell in multiple ways:

1. When exiting Yazi, change the cwd to the path when you exited. See https://yazi-rs.github.io/docs/usage/quick-start#changing-working-directory-when-exiting-yazi
2. Without exiting Yazi, open a new shell within Yazi and return to Yazi whenever you are done with your operations. See https://github.com/sxyazi/yazi/issues/177#issuecomment-1722587764
3. Press `C-z` to pause Yazi and return to the terminal; after completing your operation, use `fg` to return to Yazi again. See https://github.com/sxyazi/yazi/pull/120

> how can I change the $Editor or add another program i'd like to use

See https://yazi-rs.github.io/docs/usage/configuration/yazi#opener
